conf/services/conflationTypes.json

```diff
@@ -109,7 +109,8 @@
         "merger": "hoot::ScriptMergerCreator",
         "members": {
             "poi.ignore.type.if.name.present": "Ignore type if name present",
-            "search.radius.poi": "Search radius (m); -1 for default"
+            "search.radius.poi": "Search radius (m); -1 for default",
+            "poi.search.radii": "The search radii distance configuration for POI to POI Conflation"
```

Comment:
Do we need both of these?
If there are some POIs that might not be covered in the default POI search radii my guess is we do need it.
The question is then do the specific poi radius values take precedence over the generic search radius value if both are customized?

Reply:
We should add a unit `(m)` for the distance

Reply:
`search.radius.poi` is currently the top level override, allowing you to search for all POI types at the same distance. I'm not sure if anyone uses that option, but I left it enabled just in case. Option priority in the config described [here](https://github.com/ngageoint/hootenanny/blob/4115_poiSearchRadii/docs/algorithms/PoiToPoiConflation.asciidoc#find-match-candidates).